web/src/components/map/map-status.tsx

```
@@ -0,0 +1,42 @@
+}
+
+export function MapLoadingDisplay({ className = "" }: MapLoadingDisplayProps) {
+  return (
+    <div 
+      className={`absolute inset-0 flex items-center justify-center bg-gray-50 border border-gray-300 rounded ${className}`}
+      aria-live="polite"
+    >
+      <div className="text-center text-gray-600">
+        <div className="animate-spin rounded-full h-8 w-8 border-b-2 border-blue-600 mx-auto mb-2"></div>
+        <p className="text-sm">地図を読み込み中...</p>
+      </div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)